### ISSUE TITLE
Distro error openstack/autopilot

### ISSUE THREAD

**Issue by ladrua** [ubuntu/conjure-up#154]
Trying conjure-up openstack using MaaS 2.0 on Ubuntu 16.04, choosing Autopilot, getting error:

```
maas openstack: [ERROR] ['Creating Juju controller "local.elida" on maas/10.0.0.51', 'Bootstrapping 
model "admin"', 'Starting new instance for initial controller', 'Launching instance', 'WARNING no 
architecture was specified, acquiring an arbitrary node', 'ERROR failed to bootstrap model: cannot start 
bootstrap instance: unexpected: ServerError: 400 BAD REQUEST ({"distro_series": ["\'trusty\' is not a 
valid distro_series.  It should be one of: \'\', \'ubuntu/xenial\'."]})']
```

**Comment by battlemidget** [comment]
Do you have both xenial and trusty images imported in MAAS?

**Comment by ladrua** [comment]
No, following the official Autopilot guide (witch seems outdated btw) it only says to import 16.04.. Should I  have both?

**Comment by ladrua** [comment]
So is it worth trying at all? If so, I then need to have both trusty and xenial importet in maas?

**Comment by battlemidget** [comment]
Autopilot uses juju 1.25 behind the scenes, and it only supports up to maas 1.9 so at this time it isn't worth trying until Autopilot officially supports Juju 2.0 and MAAS 2.0 neither of which are out of beta yet.

**Comment by ladrua** [comment]
So what would you recommend then? Should I re-install 14.04 as my MaaS server to get maas 1.9 ? I had Autopilot up and running on 15.10, and it crashed. Really want it back :)

**Comment by battlemidget** [comment]
Yea I would stick with 14.04 until Autopilot announces official support for MAAS 2.0

**Comment by ladrua** [comment]
Ok, thank you for your help(again)!

**Comment by xaionaro** [comment]
Why the issue is closed? Problem is not solved :(

**Comment by battlemidget** [comment]
No problem to fix as Autopilot is not available for Juju 2.0/Maas 2.0 yet.

**Comment by moparlakci** [comment]
you need to import ubuntu 14.04 images as well
